Адаптивная сетка

AleksanrReva opened the pull request:
---
:mortar_board: [Адаптивная сетка](https://up.htmlacademy.ru/adaptive-individual/1/user/1411729/tasks/7)

:boom: https://htmlacademy-adaptive.github.io/1411729-cat-energy-1/5/

AleksanrReva committed:
добавляет viewport

AleksanrReva committed:
добавляет переменные

добавляет значения ширин для дальнейшего использования в медиа запросах

AleksanrReva committed:
стилизует меню

AleksanrReva committed:
делает главную страницу адаптивной

добавляет адаптивную сетку крупных блоков для мобильного состояния

AleksanrReva committed:
делает сетку адаптивной

добавляет сетку таблице из живого примера для планшетной версии

AleksanrReva committed:
добавляет адаптив странице с формой

добавляет адаптивную сетку крупных блоков для планшета

AleksanrReva committed:
добавляет адаптив каталогу

добавляет адаптивную сетку крупных блоков для планшета

AleksanrReva committed:
стилизует мобильный вид index

делает небольшие правки по расстояниям по всему документу
стилизует таблицу в блоке живого примера

AleksanrReva committed:
выравнивает шапку сайта

стилизует шапку под разрешение экрана планшета

AleksanrReva committed:
выравнивает контент главной страницы

стилизует содержимое main-блока и мелкие сетки под разрешение экрана планшета

AleksanrReva committed:
стилизует подвал сайта

задаёт сетку для разрешения экрана больше, либо равно 768px по ширине

AleksanrReva committed:
добавляет адаптив шапке и футеру

делает сетку шапки и футера адаптивной к разрешению экрана больше или равно 1440px

AleksanrReva committed:
добавляет сетки главной странице

делает главную страницу и отдельные её элементы адаптивной к экрану компьютера

AleksanrReva committed:
правит мобильный вид формы

добавляет нужные отступы;
стилизует инпуты

AleksanrReva committed:
стилизует страницу с формой 768px

задает размеры шрифтам;
исправляет, добавляет нужные отступы;
добавляет мелкие сетки

AleksanrReva committed:
стилизует страницу с формой 1440px

исправляет, добавляет нужные отступы;
добавляет мелкие сетки

AleksanrReva committed:
редактирует мобильный вид каталога

AleksanrReva committed:
правит и дополняет сетку каталога

делает страницу каталога адаптивной к размеру экрана планшета

AleksanrReva committed:
адаптирует каталог к разрешению монитора

делает сетку каталога адаптивной к ширине экрана более 1440px

Kartaviy-arteom commented on source/less/blocks/advantages.less:
```diff
@@ -8,6 +8,46 @@
+  }
+
+  .advantages__list {
+    grid-template-columns: repeat(2, 250px);
```
тут лучше колонки тоже тянуть grid-template-columns: repeat(2, 1fr)

AleksanrReva replied:
![Снимок экрана 2024-03-18 204327](https://github.com/htmlacademy-adaptive/1411729-cat-energy-1/assets/48528980/1a44426b-e3a7-4a11-b010-20477c973957)
У меня так едет вёрстка, что мелочи, но нужно будет тогда шириной ровнять текст, плюс там же нумерация на фоне, тоже потребует отступов. Или ок?

Kartaviy-arteom commented on source/less/blocks/site-footer.less:
```diff
@@ -3,5 +3,40 @@
 
 .site-footer__map {
-  margin-bottom: 39px;
+  margin-bottom: 40px;
```
изображение внутри должно "закрыввать весь блок"

Kartaviy-arteom commented on source/less/blocks/footer-links.less:
```diff
@@ -1,4 +1,72 @@
-.footer-links__logotype {
+.footer-links__list  {
```
будет ломаться футер при переполнениями соц сетями

Kartaviy-arteom reviewed (changes requested):
нужно исправить ошибки линтеров (npm run lint)

AleksanrReva committed:
исправляет недочёты и ошибки

исправляет переполнение соцсетями в футере;
делает изображение карты резиновым;
исправляет замечания линтера

AleksanrReva committed:
делает сетку преимуществ резиновой

делает сетку списка преимуществ резиновой для экрана планшета
